components/Button.tsx

```diff
@@ -0,0 +1,13 @@
+import React, { ButtonHTMLAttributes } from 'react';
+
+import styles from '../styles/Button.module.scss';
+
+type ButtonProps = {} & ButtonHTMLAttributes<HTMLButtonElement>;
```

Comment:
We can omit this type and just pass the button attributes in the type below. Also arrow function to keep consistency